File: app/src/main/java/com/wifiui/MainActivity.java
package com.wifiui;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView tv = new TextView(this);
        tv.setText("Hello from Java!");
        tv.setTextSize(20);
        tv.setPadding(40, 40, 40, 40);
        setContentView(tv);
    }
}

File: app/src/main/kotlin/com/hhh/MainActivity.kt
package com.hhh

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val tv = TextView(this)
        tv.text = "Hello from Kotlin!"
        tv.textSize = 20f
        tv.setPadding(40, 40, 40, 40)
        setContentView(tv)
    }
}
